internal/client/v1alpha1/informers/externalversions/ako/v1alpha1/interface.go

```diff
@@ -24,6 +24,8 @@ import (
 
 // Interface provides access to all the informers in this group version.
 type Interface interface {
+	// AlbInfraSettingses returns a AlbInfraSettingsInformer.
+	AlbInfraSettingses() AlbInfraSettingsInformer
```

Comment:
`AlbInfraSettingses` looks weird, I think we need to keep the CRD name singular.

Reply:
from the crd yaml
```
  names:
    kind: NsxAlbInfraSetting
    listKind: NsxAlbInfraSettingList
    plural: nsxalbinfrasettings
    shortNames:
    - nsxalbinfrasettings
    - nas
    singular: nsxalbinfrasetting
```
I think the struct names need to change to Singular in `albinfrasettings.go`
We also need to make it consistent everywhere, either `nsxalb...` or just `alb...`

Reply:
Ack. Done.